projects/core/components/root/root.style.less

```diff
@@ -7,19 +8,28 @@
     }
 }
 
-:host {
+html,
+.tui-zero-scrollbar {
+    .scrollbar-hidden();
+}
+
+body {
+    .text-basic();
+}
+
+tui-root {
     position: relative;
     display: block;
     flex: 1;
     -webkit-tap-highlight-color: transparent;
 }
 
-.t-scrollbar {
+tui-root .t-root-scrollbar {
```

Comment:
`>` if you want to be safe, otherwise let's just drop `tui-root`.

Reply:
it doesn't work without tui-root, I added `>`